doc/getting_started/contributing.rst

```diff
@@ -125,7 +155,7 @@ You can see the result by pointing your browser to::
     doc/_build/latex/Refl1d.pdf
 
 Note that this only works with a unix-like environment for now since we are
-using *make*.  On windows, you can run sphinx directly from python::
+using *make*.  On Windows, you can run sphinx directly from python::
```

Comment:
I think make is pretty easy to install from conda-forge now... might simplify things on Windows, though I haven't tested it for building the documentation.  Might be additional dependencies to install as well.